packages/open-payments/src/client/incoming-payment.test.ts

```diff
@@ -74,6 +77,21 @@ describe('incoming-payment', (): void => {
         method: HttpMethod.POST
       })
     })
+
+    test('creates listIncomingPaymentsOpenApiValidator', async (): Promise<void> => {
```

Comment:
@raducristianpopa can we try testing that the `list` route calls the proper validator, similar to this test? https://github.com/interledger/rafiki/pull/848/files#diff-4f318c2a374cc145f8825fecf95d77b29a728927b5a64ce393897b01becb2b93R37-R38

If it takes too long, I'm going to approve the PR anyway so we can merge it later and working on these route tests at a separate time